src/Token/AccessToken.php

```diff
@@ -84,6 +89,17 @@ public function __construct(array $options = [])
 
             $this->expires = $expires;
         }
+
+        // Capure any additional values that might exist in the token but are
+        // not part of the standard response. Vendors will sometimes pass
+        // additional user data this way.
+        $this->values = array_diff_key($options, array_flip([
+            'access_token',
+            'resource_owner_id',
+            'refresh_token',
+            'expires_in',
+            'expires',
+        ]));
```

Comment:
Do you think it would be good for us to put our standard keys in a property or something? Ideally, this would be an array class constant, but we would need to enforce PHP >= 5.6.0 for that, and I don't think we should jump to that just yet. :-)